### ISSUE TITLE
inference script not working

### ISSUE THREAD

**Issue by ncdejito** [Thinklab-SJTU/R3Det_Tensorflow#31]
applied tools/inference.py on DOTA Test set 2 images, still no bounding boxes. Adjusted libs/configs/cfgs.py VIS_SCORE to 0.0, still no bounding boxes.

Here's the code I used
`python inference.py --data_dir '/dir/of/2/test/images/' --save_dir '/my/chosen/folder/' --gpu 0 --draw_imgs`

Here's the output running inference script on DOTA Test set P0034.png
![image](https://user-images.githubusercontent.com/13001116/83540675-35063f00-a52b-11ea-8b25-e6d6e1887b42.png)

_Originally posted by @ncdejito in https://github.com/Thinklab-SJTU/R3Det_Tensorflow/issues/7#issuecomment-637639110_

**Comment by ncdejito** [comment]
weights and folder locations
![image](https://user-images.githubusercontent.com/13001116/83542632-ceceeb80-a52d-11ea-8183-4f5b8020d5cb.png)

**Comment by yangxue0827** [comment]
![image](https://user-images.githubusercontent.com/29257168/83543730-70a30800-a52f-11ea-9270-1f5c66305286.png)
infinference.py is used for small images. @ncdejito

**Comment by ncdejito** [comment]
tried test_dota_r3det_plusplus.py, still no bounding boxes.
```
cd ~/R3Det_Tensorflow/tools
python test_dota_r3det_plusplus.py --test_dir='/dir/of/2/test/images/' --gpus=0 --show_box
cd test_dota/RetinaNet_DOTA_R3Det_2x_20200501/dota_img_vis
```

**Comment by ncdejito** [comment]
ran with gpus param 0-7, still no bounding boxes.
```
cd ~/R3Det_Tensorflow/tools
python test_dota_r3det_plusplus.py --test_dir='/dir/of/2/test/images/' --gpus=0,1,2,3,4,5,6,7 -s
cd test_dota/RetinaNet_DOTA_R3Det_2x_20200501/dota_img_vis
```

**Comment by ncdejito** [comment]
python: 3.5.6
cuda: 10.0.130
cv2: 4.2.0
tfplot: 0.2.0
tf: 1.13.2

OS: Ubuntu 16.04

**Comment by ncdejito** [comment]
tried cropping to 600x600 images, still no bounding boxes
![image](https://user-images.githubusercontent.com/13001116/83912413-6b98bf80-a7a0-11ea-9d41-126f45ffef90.png)

**Comment by yangxue0827** [comment]
![image](https://user-images.githubusercontent.com/29257168/83965187-5dec5280-a8e4-11ea-9f35-d1209dc45a49.png)
This line of code needs to be adjusted according to the model you use
![image](https://user-images.githubusercontent.com/29257168/83965208-7ceae480-a8e4-11ea-9c30-e206768e0ecb.png)

**Comment by gaochuan2017** [comment]
> ![image](https://user-images.githubusercontent.com/29257168/83965187-5dec5280-a8e4-11ea-9f35-d1209dc45a49.png)
> This line of code needs to be adjusted according to the model you use
> ![image](https://user-images.githubusercontent.com/29257168/83965208-7ceae480-a8e4-11ea-9c30-e206768e0ecb.png)

I have met the same question,and the picture cannot be opened so I cannot see the solution in the picture, would you please describe how can I fix this bug? Thanks!

**Comment by kevjamco** [comment]
Hello, I'm having the same issue as above. For any config file I use, I get no predictions. I'm using the docker image, the DOTA training set, and cropped images. (1024x1024). Anyone have any luck getting this working?

**Comment by rampage-rider** [comment]
I am facing a similar Issues when trying to run the **ResNet152_v1d MS** model

I have loaded the pre-trained model in the `output/trained_weights` as shown below
<img width="315" alt="trained weight struct" src="https://user-images.githubusercontent.com/56698590/88832026-cc1b0a80-d1ed-11ea-9f73-2b96e3182349.png">

I have also loaded the pre-trained weights in the `data/pretrained_weights` as shown below
<img width="322" alt="Pre-weights models tree struct" src="https://user-images.githubusercontent.com/56698590/88832085-e81eac00-d1ed-11ea-8e66-6e592d220f94.png">

I have also modified the `cfgs.py` code with the `cfgs_res152_dota_r3det_plusplus_v1.py` code.

I execute the code using `python test_dota_r3det_plusplus_ms.py --test_dir='../data/dataset/test/' --gpus=0 -s`

The model runs, but the output images have no bounding boxes. Am I doing something wrong?

**Comment by yangxue0827** [comment]
![image](https://user-images.githubusercontent.com/29257168/88834238-1d36f880-d206-11ea-8865-7c867e495fd4.png)

Need to be modified to the following

`from libs.networks import build_whole_network_r3det_plusplus`

**Comment by rampage-rider** [comment]
Yes I have made the modification in `test_dota_r3det_plusplus_ms.py` file
![image](https://user-images.githubusercontent.com/56698590/88834982-3c2b8f80-d1f2-11ea-9cf5-08c20e1be3a2.png)

Is there any other file in which this modification needs to be done?
